spec/feature/user/creates_a_listing_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+describe 'user creates a listing', type: :feature do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.